OnlineStore.UI/Controllers/OrderController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OnlineStore.BLL.DTO;
+using OnlineStore.BLL.Services;
+using OnlineStore.BLL.Services.Interfaces;
+
+namespace OnlineStore.UI.Controllers
+{
+	[Route("api/[controller]")]
+	[ApiController]
+	public class OrderController : ControllerBase
+	{
+		private readonly IOrderService _orderService;
+		public OrderController(IOrderService orderService)
+		{
+			_orderService = orderService;
+		}
+
+		[HttpPost]
+		[ProducesResponseType(StatusCodes.Status201Created)]
+		[ProducesResponseType(StatusCodes.Status400BadRequest)]
+		public async Task<ActionResult<CreateOrderDto>> CreateOrderAsync([FromBody] CreateOrderDto orderDTO, CancellationToken cancellationToken)
+		{
+			if (orderDTO == null)
```

Comment:
i think lines 24-27 are redundant here as if dto is null there will be automatically send 400 code